caffe2/contrib/aten/aten_op_template.h

```diff
@@ -179,7 +180,8 @@ class ATenOp : public Operator<Context> {
     std::vector<std::string> attrs;
     for (const auto i : c10::irange(operator_def.arg_size())) {
       auto & attr = operator_def.arg(i);
-      if(attr.name() == "operator" || attr.name() == "type" || attr.name() == "overload_name" ) {
+      if (attr.name() == "operator_name" || attr.name() == "type" ||
```

Comment:
it's not backward compatible. If there are some models with serialized Caffe2 Nets that contain ATenOp it'd break. You'd need either:
1) make this op recognize both types of arguments, i.e. `operator_name` and `operator`
2) change the lowering from ONNX to Caffe2 to rename the arg back to be just `operator`

I probably prefer the second one

Reply:
I have refactored Caffe2 code to accept such change and verified using all Unit Tests available to make sure this is not BC breaking

Reply:
Verifying that unittests pass is not enough. If there's a serialized Caffe2 NetDef model somewhere with an ATen op inside, it won't be openable with the new code. We don't have unittest that exercises the serialized Caffe2 model afaiu.

At least at Meta we still have a few models like that laying around. So I'd prefer you not to change Caffe2's op signature. But since you have PT->ONNX->C2 two step conversion, you can evolve ONNX's operator spec as you fit while keeping the Caffe2 op spec intact for the existing models.

The similar change you pointed out is not affecting how already converted models are loaded, so it's ok.

Reply:
Reverting to `operator`. Thank you for the heads up